### PR TITLE
Support group deregistration

### PR DESCRIPTION
**Pull request by eddelbuettel**
This PR adds a (still somewhat minimal) group deregistration function as a 'manual' overlay function.

To support it, code generation was re-run -- but only newly-created group* files were carried over into directory R/ along with their help files into man/.  

CI was changed to not fail.  For some reason, even with the decoded login, the server always denies. So Linux now mirrors macOS and does not run them at Azure making Azure turn green.

### CODE REVIEW

File: R/manual_layer_group_deregister.R [hunk 1]
```diff
@@ -0,0 +1,64 @@
+
+##' De-register a 'Group' object recursively
+##'
+##' This function de-registers a 'Group' object, the 'Group' objects therein
+##' as well as any 'arrays.
```
Comment (johnkerl) [inline R/manual_layer_group_deregister.R:5]: `'arrays` -> `'arrays'` ?

**Review by johnkerl — APPROVED** [review]
🎆 🚀 
Awesome!! :D

File: R/manual_layer_group_deregister.R (hunk 1 above)
Comment (eddelbuettel) [reply R/manual_layer_group_deregister.R:5]: Thanks -- will fix to no apostrophes:   "as well as any arrays."

**Review by awenocur — APPROVED** [review]

**Comment by ihnorton** [comment]
Thanks @eddelbuettel 💯

> CI was changed to not fail. For some reason, even with the decoded login, the server always denies. So Linux now mirrors macOS and does not run them at Azure making Azure turn green.

I think we need to dig in to this a bit, because right now it's not clear what level of testing this leaves us with on CI. My first guess would be that the .enc file contains an expired token, but I don't have access to the repo settings in order to investigate.

Alternatively, if we can commit to fixing it in the course of making a switch to github actions as an immediate follow-up PR, then IMO it's not necessarily blocking.

**Comment by eddelbuettel** [comment]
Yes. I want to add one (local) test as well -- and we should probably look into why it stopped working. Which may require a concurrent look at server logs.

**Comment by eddelbuettel** [comment]
Turns out adding a test is not as straightforward because ... using the `tiledbcloud` package, even on filesystem-local URI, requires a `login()` step.

**Comment by eddelbuettel** [comment]
>  My first guess would be that the .enc file contains an expired token.

I concur. I just re-traced all the steps (without being able to decode, of course).

**Comment by eddelbuettel** [comment]
New token issued, and setup and tested in a separate (smaller) scrap repo.  

This now needs @Shelnutt2 to either a) add the token to a 'secret' environment variable at Azure DevOps (if we want to keep it here) or b) add it to a repository secret for GitHub Actions here.

**Comment by eddelbuettel** [comment]
So with the slight detour of #114 rebased in we now have the desire green hue all around here from approvals and checks. 

Good to :shipit: then I presume?

**Review by ihnorton — APPROVED** [review]

**Pull request merged**